example/android/app/build.gradle

```diff
@@ -27,6 +27,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
+    namespace 'com.example.amplitude_flutte_example'
```

Comment:
nit: flutte**r**

Reply:
Nice catch! Update in https://github.com/amplitude/Amplitude-Flutter/pull/172/commits/5c4ad5998af32fb46d826192f946b66e3bc03d82